.circleci/config.yml

```diff
@@ -61,6 +61,17 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v[0-9].[0-9].[0-9]+.*/
+      - publish-dev:
+          requires:
+            - cross-compile
+            - loadtest
+            - test
+            - coverage
+          filters:
+            branches:
+              only: master
+            tags:
+              ignore: /.*/
```

Comment:
Do I read this correctly that you want any tag here? And above  `/^v[0-9].[0-9].[0-9]+.*/` all `v.1.2.3anythinghafter` so probably the above should not allow anything after the version, also this tag should probably be /^v[0-9].[0-9].[0-9]+-dev/

Reply:
Currently for a tag that matches stable we do two releases which is probably not expected.

Reply:
So I think we should fix this.

Reply:
I'll update the regex to match tags more precisely. 

> Currently for a tag that matches stable we do two releases which is probably not expected.

Not sure I follow this.